components/server/ee/src/workspace/gitpod-server-impl.ts

```diff
@@ -2649,6 +2649,11 @@ export class GitpodServerEEImpl extends GitpodServerImpl {
 
         const context = (await this.contextParser.handle(ctx, user, contextURL)) as CommitContext;
 
+        // HACK: treat manual triggered prebuild as a reset for the inactivity state
+        await this.projectDB.updateProjectUsage(project.id, {
```

Comment:
WDYT, should we make the state to `Project.state`?

Reply:
If possible, I'd love to stay away from very generic attribute names like "state" (which can mean almost anything 🙈).

Also, personally I don't see a need to change the current separate table, but would also be happy with this becoming a `Project.lastRecordedActivityTime` field or so.